Python scripts/Turtle_Experiment3DShapes.py

```diff
@@ -0,0 +1,32 @@
+
+# coding: utf-8
+
+# In[ ]:
+
+
+#Importing modules
+import turtle
+import random
+
+#defining turtle
+t = turtle.Pen()
+turtle.bgcolor("black")
+t.pencolor("purple")
+
+#Deciding how many times it will be run and the x,y center location shapes
+for n in range(50):
+    x = random.randrange(-turtle.window_width()//2,
```

Comment:
Using this double slash `//` made me go and look up what it did in Python. Learnt today that it floors the number rather than comments it. Handy little feature!